### PR TITLE
Update basic tutorial for docs (TEMPLATE)

### PR DESCRIPTION
**Pull request by stephanielee9**
Created a new basics folder for intro tutorial and updated separate `css` and `qss` notebooks. 

To update old examples notebooks:
* Add descriptions so new users have an easier time following code in all relevant files (e.g., `.rst` file and two notebooks).
* The title (at `h1` level) for each notebook should be the first cell (e.g., `#Title`). No other titles can be at this level or the sidebar will look funny. Subtitles can be `h2`, `h3`, etc. (e.g., `##Subheading`, `###Subheading`). 
* As much as possible, the `css` and `qss` notebooks should look identical.
* Build the docs to see how things look (i.e., `make clean`, `make html`).

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -42,7 +42,7 @@ Learn more about SuperstaQ `here <https://www.super.tech/about-superstaq/>`_.
 
    get_started/credentials
    get_started/installation
-   get_started/basics
+   get_started/basics/basics
```
Comment (vtomole) [inline docs/source/index.rst:45]: `basics/basics`

Redundant

Comment (stephanielee9) [reply docs/source/index.rst:45]: The `basics.rst` file is in the `basics` directory. Unless you're referring to something else?

Comment (vtomole) [reply docs/source/index.rst:45]: Ah okay. Sounds good.

File: docs/source/get_started/basics/basics_css.ipynb [hunk 2]
```diff
@@ -41,13 +38,8 @@
...
     "import cirq_superstaq as css\n",
     "\n",
-    "# requirements to use qiskit-superstaq\n",
-    "import qiskit\n",
-    "import qiskit_superstaq as qss\n",
-    "\n",
     "# optional\n",
-    "import os  # used to set a token as an environment variable\n",
-    "import ipywidgets as widgets  # not required to use SuperstaQ; used here for interactive demonstrations"
+    "import os  # used to set a token as an environment variable"
```
Comment (dowusu-antwi) [inline docs/source/get_started/basics/basics_css.ipynb:42]: nit: wording
```suggestion
    "import os  # used to store a token in an environment variable"
```

File: docs/source/get_started/basics/basics_css.ipynb [hunk 3]
```diff
@@ -59,12 +51,15 @@
...
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "e4d4ae45",
    "metadata": {},
    "source": [
-    "## Set up access to SuperstaQ's API\n",
-    "You will need to first grab your SuperstaQ API key. Here, we set the API key as an environment variable, but you may also set your token in the `api_key` argument of `css.Service()` (if using cirq-superstaq) or `qss.SuperstaQProvider()` (if using qiskit-superstaq)."
+    "## Set up access to Superstaq's API\n",
+    "You will need to first grab your Superstaq API key. Here, we set the API key as an environment variable, but you may also set your token in the `api_key` argument of `css.Service()`.\n",
```
Comment (dowusu-antwi) [inline docs/source/get_started/basics/basics_css.ipynb:60]: nit: wording
```suggestion
    "You will need to first grab your Superstaq API key. Here, we store the API key in an environment variable, but you may also pass your token to the `api_key` parameter of `css.Service()`.\n",
```

File: docs/source/get_started/basics/basics_css.ipynb [hunk 4]
```diff
@@ -59,12 +51,15 @@
...
+    "\n",
+    "To set an environment variable, run `export SUPERSTAQ_API_KEY=\"<token>\"`, where `token` is the token API key you have retrieved from https://superstaq.super.tech, in the terminal where you are using Superstaq. Alternatively, you can add a cell to this notebook and run `!export SUPERSTAQ_API_KEY=\"<token>\"`."
```
Comment (dowusu-antwi) [inline docs/source/get_started/basics/basics_css.ipynb:62]: nit: wording
```suggestion
    "To set up an environment variable, run `export SUPERSTAQ_API_KEY=\"<token>\"`, where `token` is the API key you have retrieved from https://superstaq.super.tech, in the terminal where you are using Superstaq. Alternatively, you can add a cell to this notebook and run `!export SUPERSTAQ_API_KEY=\"<token>\"`."
```

File: docs/source/get_started/basics/basics_qss.ipynb [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+    "# optional\n",
+    "import os  # used to set a token as an environment variable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0d0a342",
+   "metadata": {},
+   "source": [
+    "If you are not using our [Hub](https://hub.super.tech/) or do not have the requirements already install on your machine, you may add a code cell into the notebook to run `!pip install <package_name>` or run in terminal `pip install <package_name>`."
```
Comment (dowusu-antwi) [inline docs/source/get_started/basics/basics_qss.ipynb:50]: typo:
```suggestion
    "If you are not using our [Hub](https://hub.super.tech/) or do not have the requirements already installed on your machine, you may add a code cell into the notebook to run `!pip install <package_name>` or run in terminal `pip install <package_name>`."
```

File: docs/source/get_started/basics/basics_qss.ipynb [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "e4d4ae45",
+   "metadata": {},
+   "source": [
+    "## Set up access to Superstaq's API\n",
+    "You will need to first grab your Superstaq API key. Here, we set the API key as an environment variable, but you may also set your token in the `api_key` argument of `qss.SuperstaQProvider()`.\n",
```
Comment (dowusu-antwi) [inline docs/source/get_started/basics/basics_qss.ipynb:60]: nit: wording
```suggestion
    "You will need to first grab your Superstaq API key. Here, we store the API key in an environment variable, but you may also pass your token to the `api_key` parameter of `qss.SuperstaQProvider()`.\n",
```

File: docs/source/get_started/basics/basics_qss.ipynb [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+    "\n",
+    "To set an environment variable, run `export SUPERSTAQ_API_KEY=\"<token>\"`, where `token` is the token API key you have retrieved from https://superstaq.super.tech, in the terminal where you are using Superstaq. Alternatively, you can add a cell to this notebook and run `!export SUPERSTAQ_API_KEY=\"<token>\"`."
```
Comment (dowusu-antwi) [inline docs/source/get_started/basics/basics_qss.ipynb:62]:
```suggestion
    "To set up an environment variable, run `export SUPERSTAQ_API_KEY=\"<token>\"`, where `token` is the API key you have retrieved from https://superstaq.super.tech, in the terminal where you are using Superstaq. Alternatively, you can add a cell to this notebook and run `!export SUPERSTAQ_API_KEY=\"<token>\"`."
```

**Review by dowusu-antwi — APPROVED** [review]
LGTM, pending mostly wording nits:

**Pull request merged**